src/main/java/org/springframework/social/bitbucket/api/impl/UsersPrivilegesTemplate.java

```diff
@@ -0,0 +1,70 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.social.bitbucket.api.impl;
+
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.social.bitbucket.api.BitBucketTeamPrivilege;
+import org.springframework.social.bitbucket.api.UsersPrivilegesOperations;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author Cyprian Śniegota
+ * @since 2.0.0
+ */
+public class UsersPrivilegesTemplate extends AbstractBitBucketOperations implements UsersPrivilegesOperations {
+    public UsersPrivilegesTemplate(RestTemplate restTemplate, boolean authorized) {
+        super(restTemplate, authorized, V1);
+    }
+
+    @Override
+    public final Map<String, BitBucketTeamPrivilege> getPrigilegeGroupOnTeamAccount(String accountName, String groupSlug) {
+        return getRestTemplate().getForObject(buildUrl("/users/{accountname}/privileges"), TeamPrivilegeHolder.class, accountName);
+    }
+
+    @Override
+    public final BitBucketTeamPrivilege getPrivilegesAssociatedWithGroup(String accountName, String owner, String groupSlug) {
+        return getRestTemplate()
+                .getForObject(buildUrl("/users/{accountname}/privileges/{owner}/{group_slug}"), TeamPrivilegeHolder.class, accountName, owner, groupSlug)
+                .get("privilege");
+    }
+
+    @Override
+    public final Map<String, BitBucketTeamPrivilege> updateGroupPrivilegesOnTeamAccount(String accountName, String owner, String groupSlug,
+            BitBucketTeamPrivilege privilege) {
+        return getRestTemplate().exchange(buildUrl("/users/{accountname}/privileges/{owner}/{group_slug}"), HttpMethod.PUT,
+                new HttpEntity<>("privileges=" + privilege.toString()), TeamPrivilegeHolder.class, accountName, owner, groupSlug).getBody();
+    }
+
+    @Override
+    public final Map<String, BitBucketTeamPrivilege> postNewPrivilege(String accountName, String owner, String groupSlug, BitBucketTeamPrivilege privilege) {
+        return getRestTemplate()
+                .postForObject(buildUrl("/users/{accountname}/privileges/{owner}/{group_slug}"), new HttpEntity<>("privileges=" + privilege.name()),
+                        TeamPrivilegeHolder.class, accountName, owner, groupSlug);
+    }
+
+    @Override
+    public final void removePrivilegeGroup(String accountName, String owner, String groupSlug) {
+        getRestTemplate().delete(buildUrl("/users/{accountname}/privileges/{owner}/{group_slug}"), accountName, owner, groupSlug);
+    }
+
+    private static final class TeamPrivilegeHolder extends HashMap<String, BitBucketTeamPrivilege> {
+        private static final long serialVersionUID = 1L;
```

Comment:
General comment: we should remove the FindBugs rule about `serialVersionUID`. The compile time generation of `serialVersionUID` is better in most cases (unless you actually want to serialize those object in a long term storage and manage versioning finely).